### PR TITLE
DCA-8: Set up viewer and viewer-plus access for DCA prod' account

### PR DESCRIPTION
**Pull request by brucehoff**
This PR is step 9 of [the account set up](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/745373697/Setup+AWS+Accounts) for the DCA prod account.

### CODE REVIEW

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -1435,3 +1443,37 @@ SsoRecoverProdViewer:
...
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref DCAProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref dcaProdViewersPlusGroup
+    permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-permission-set-arn' ]
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:1479]: you probably want viewer plus permission set here?

File: org-formation/700-aws-sso/_tasks.yaml [hunk 2]
```diff
@@ -225,6 +225,14 @@ Parameters:
     Type: String
     Default: 'b4e86468-f091-70e2-0ea9-64cf3addcdf1'
 
+  dcaProdViewersGroup:      #JC aws-dca-prod-viewers
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:228]: no DCA team member needs admin or developer role for this account?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml:228]: Correct.  We will follow up with a new role that gives app' creators permission to manage secrets (see https://sagebionetworks.jira.com/browse/DCA-35) but not have administrative access.

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml]: Yes, thanks.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**